types/chrome/index.d.ts

```diff
@@ -7184,25 +7184,50 @@ declare namespace chrome.storage {
         getBytesInUse(callback: (bytesInUse: number) => void): void;
```

Comment:
From what I can tell this definition is useless -- It's technically OK, but will always return 0 in the callback. 
What do you think about removing lines :7179 through :7984? Please LMK if you think I'm off-base, though!

Reply:
The problem here if this is removed is the following:

`chrome.storage.sync.getBytesInUse(() => {});` which should be valid gives error because it is taking the first type which returns promise and accepts keys as an argument.

Reply:
Ah - I didn't think this would cause an error with just `.d.ts` files. Thanks, and sorry for sending you astray!